lighthouse-core/gather/gather-runner.js

```diff
@@ -616,6 +628,8 @@ class GatherRunner {
       // @ts-expect-error - guaranteed to exist by the find above
       baseArtifacts.NetworkUserAgent = userAgentEntry.params.request.headers['User-Agent'];
     }
+
+    baseArtifacts.FullPageScreenshot = await this.getFullpageScreenshot(passContext);
```

Comment:
Should we try to not run this if no audit requires `FullPageScreenshot`? if so, how to wire that here?

Reply:
Is this significantly slower than other base artifacts?